src/CoreBundle/Resources/translations/admin.en.xliff

```diff
@@ -2606,6 +2606,11 @@
                 <source>chameleon_system_core.navigation_tree.cancel_selection</source>
                 <target>Cancel</target>
             </trans-unit>
+
+            <trans-unit id="chameleon_system_core.record_list.sql_error">
+                <source>chameleon_system_core.record_list.sql_error</source>
+                <target>A database error occurred creating the record list. Most likely, this is a configuration error in table settings. The error has been logged.</target>
```

Comment:
Isn't this "most likely" part a bit too specific?

I think "A database error occurred while creating the record list." would be sufficient.

That it is logged (and thus there are more informations to be found) should be general consensus for errors, I guess.

Reply:
My experience is, that errors are usually not logged. Including the information that is actually is logged here, could potentially  save someone quite some time. 
I don't know if a more general message would be enough to immediately make you think of the table settings and I don't see the advantage of a less specific message?